otter/grade/__init__.py

```diff
@@ -117,7 +123,8 @@ def main(
 
     pdf_dir = os.path.join(output_dir, "submission_pdfs") if pdfs else None
 
-    grade_dfs = launch_containers(
+    monitor_progress(result_queue, print_progress, "Launching Grading Process")
```

Comment:
Again, please make use of the existing logging infrastructure by logging to `LOGGER`. If you want logs to also be written to the queue, I'd suggest creating a new logging handler that does this.